packages/create-hint/src/templates/locales-messages.json.hbs

```diff
@@ -0,0 +1,12 @@
+{
+    {{#each hints}}
+    "{{#if ../isMulti}}{{normalizedName}}_{{/if}}description": {
```

Comment:
We are not using the `normalizedName` here. We are using the `camelcase`, I think we already have a helper for that "toCamelCase"

Reply:
You can see an example here: https://github.com/webhintio/hint/blob/master/packages/hint-typescript-config/src/_locales/en/messages.json

Reply:
Yes right. good catch. Thanks. I didn't see for that long names.

So should I use `toCamelCase` helper on `normalizedName` or any other name?

Reply:
I got it. It is helper registered there...